list.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2014,2015,2016,2017 Docker, Inc.
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"text/tabwriter"
+	"time"
+
+	"encoding/json"
+
+	"github.com/urfave/cli"
+
+	vc "github.com/containers/virtcontainers"
+)
+
+const (
+	formatOptions = `table or json`
+
+	// ociConfigPathKey is the annotation key to fetch the OCI configuration file path.
+	ociConfigPathKey = "com.github.containers.virtcontainers.pkg.oci.config_path"
```

Comment:
It is not a good thing to duplicate such a const, because we have to make sure it is always sync'ed with virtcontainers. I think we should export this in the containers/virtcontainers/pkg/oci and use it directly from here.